packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -5,6 +5,8 @@ const directoryPath = path.resolve(__dirname, '../../../../node_modules/executio
 const axios = require('axios');
 const openRpcData = require('../../../../docs/openrpc.json');
 require('dotenv').config();
+import Ajv from 'ajv';
+import addFormats from 'ajv-formats';
```

Comment:
my editor says it cannot find this module, ajv-formats.

<img width="861" alt="image" src="https://github.com/user-attachments/assets/13f3db1a-4769-4f95-a86d-f7fd02097d21">

Reply:
Honestly same, however this is installed and its in the package.json, not sure why it complains @quiet-node